electron/src/install-dependencies.sh

```diff
@@ -3,4 +3,7 @@
 set -e -o pipefail
 
 npm install
-#./node_modules/.bin/electron-rebuild --only="node-hid"
+# ./node_modules/.bin/electron-rebuild --only="node-hid"
```

Comment:
What are these extra lines for?

Reply:
These lines are for compiling the native dependencies used by the js library, but the js library is deactivated. I did not remove the code just as a precaution in case an important unforeseen problem forces us to return to the js library, but the idea is to remove it later.

Reply:
Can you add a comment explaining why the code is commented out and not just deleted

Reply:
Done, in c05a202. I added comments indicating that the code is still there just for precaution and with some instructions about how to reactivate it if nedded.